Source/Engine/Render2D/Render2D.cpp

```diff
@@ -795,6 +812,25 @@ void Render2D::PopClip()
     OnClipScissors();
 }
 
+void Render2D::PushTint(const Color& tint)
+{
+    RENDER2D_CHECK_RENDERING_STATE;
+    
+    TintLayersStack.Push(tint);
```

Comment:
Should nested tints override the current tint or should they be combined?

Reply:
I think it should just override the previous one. Combining multiple tints would give dark results. But i can add it if needed.

Reply:
Yeah, let's make it inherited from the current tint stack even if it would make a dark result. For instance, one container could use it to fade pixels to back a little, and another child would use alpha tint to be less transparent so in the inherited case it will work fine.
Simply multiple tint by the current tint from the stack when adding it there.

Reply:
Pushed 🥳